src/main/java/org/mastodon/mamut/launcher/RecentProjectsPanel.java

```diff
@@ -128,24 +128,24 @@ private void remakeGUI( final Consumer< String > projectOpener )
 				gbc.gridy++;
 				gbc.gridx = 0;
 				final JTextArea ta = new JTextArea( projectPath );
-				ta.setEditable( false );
-				ta.setLineWrap( true );
-				ta.addMouseListener( new MouseDblClickOpenPath( projectPath ) );
-				add( ta, gbc );
```

Comment:
Please use this code formatting for Mastodon source code: https://github.com/scijava/scijava-coding-style/blob/master/src/main/resources/eclipse-formatter-settings/imglib2-coding-style.xml
The should be a space after each opening parenthesis "( " and before each closing parenthesis " )".

Reply:
Okay. I changed the coding style template to this one.
I have to confess, I do not particularly like these extra spaces, but it will be just a matter of getting used to it. So, no problem with it from my side.